conda_tutorial.md

```diff
@@ -0,0 +1,184 @@
+# Environment management with Conda
+
+## Learning objectives
++ Why should I use a package and environment management system as part of my research workflow?
++ What is Conda? Why use Conda?
++ Manage Conda environments
++ Conda channels & packages
+
+# Why should I use a package and environment management system?
+
+![](_static/soft_prob.png)
+
+Installing software is hard. Installing scientific software (including all required dependencies of said software!) is often even more challenging. 
+
+Installing software system-wide  creates complex dependencies between your reearch projects that shouldn’t really exist!
+
+Rather than installing software system-wide, wouldn’t it be great if we could install software separately for each research project?
+
+
+
+# What is Conda?
+
+Conda is open source package and runs on Windows, Mac OS and Linux.
+
++ Conda can quickly install, run, and update packages and their dependencies.
++ Conda can create, save, load, and switch between project specific software environments on your local computer.
+
+![](_static/conda1.png)
+
+Conda as a package manager helps you find and install packages. If you need a package that requires a different version of Python, you do not need to switch to a different environment manager, because Conda is also an environment manager. With just a few commands, you can set up a totally separate environment to run that different version of Python, while continuing to run your usual version of Python in your normal environment.
+
+
+# How does Conda work
+![](_static/conda2.png)
+
+# Benefits of Conda
+
+![](_static/conda3.png)
+
+
+
+## Installing & Activating Conda
+We have already installed conda on this instance, but please see the [installation HackMD](https://hackmd.io/im0eDxViQgmxTL4zYBfGoQ) after the class if you'd like to do this yourself.
+
+To be able to use that installation, we need to let the instance know what path to find bioconda in:
+
+```{bash}
+echo export PATH=$PATH:/opt/miniconda3/bin >> ~/.bashrc
```

Comment:
this should happen automatically in the conda installation process...does it not?